types/nightwatch/index.d.ts

```diff
@@ -7468,6 +7459,3 @@ export interface PageObjectModel {
     commands?: any;
     props?: any;
 }
-
-declare const _default: Nightwatch;
-export default _default;
```

Comment:
All the changes to this package were motivated by this being incorrect, but I had to fix and stub a lot more stuff to correct it. The classes I added have some `any`s where it wasn’t obvious what the types were from the implementation source code, but they are complete member-wise.